src/main/java/controllers/MySQLController.java

```diff
@@ -60,6 +60,110 @@ public void editPost(Post newPost) {
             e.printStackTrace();
         }
     }
+    /**
+     * Given an user objects changes all the attributes
+     * such that match the ones given in the parameter
+     * except the id
+     * @param user user object that stores the id to
+     *             update the values
+     */
+    public boolean updateUser(User user){
+        try {
+            String query = "UPDATE FROM `user_info` SET `username`= ?, `password`=?, `bio`=? " +
+                    "WHERE `username`=?";
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setString(1, user.getUsername());
+            preparedStatement.setString(2, user.getPassword());
+            preparedStatement.setString(3, user.getBio());
+            preparedStatement.setString(4, user.getUsername());
+            preparedStatement.execute();
+            return true;
+
+        } catch (Exception e){
+            e.printStackTrace();
+            return false;
+        }
+    }
+
+    /**
+     * deletes all the information except for recipes related
+     * to the given user in the database
+     * @param user user object that contains information about
+     *             the user
+     * @return true if it could delete all the information related
+     * to the user
+     */
+    public boolean deleteUser(User user){
+        try {
+            String query = "DELETE FROM `user_info` WHERE `user_id`=?";
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setString(1, user.getId());
+            preparedStatement.execute();
+
+        } catch (Exception e){
+            e.printStackTrace();
+            return false;
+        }
+        this.deletePostsFromUser(user);
```

Comment:
If these methods are one-liners that just call deletingFromUser, I think we could get rid of these and just call deletingFromUser directly.